vortex-array/src/array/null/compute.rs

```diff
@@ -22,6 +26,18 @@ impl ComputeVTable for NullEncoding {
     }
 }
 
+impl BinaryNumericFn<NullArray> for NullEncoding {
```

Comment:
I don't think Null arrays support numeric operations. This isn't nullable int, this is literally only null. Let's remove it